src/components/templates/PartyMap/components/RoomAttendance.tsx

```diff
@@ -35,10 +40,12 @@ export const RoomAttendance: FC<PropsType> = ({ venue, room }) => {
                 style={{ backgroundImage: `url(${user.pictureUrl})` }}
               />
             )}
-            <div></div>
+            {/* TODO: do we need this empty div here? If so, why? */}
+            <div />
```

Comment:
- [ ] [TODO] do we need this empty `div` here? If so, why?

Reply:
I think that is related to our CSS styling or similar

Reply:
removed it in https://github.com/sparkletown/sparkle/pull/840/commits/881676de95fe02cb743898b3bff51818292514fa

if it breaks things, we can re-add it